src/components/themed/TransactionListTop.tsx

```diff
@@ -110,10 +121,13 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
           <View style={styles.balanceBoxBalanceContainer}>
             <View style={styles.balanceBoxWalletNameCurrencyContainer}>
               <TouchableOpacity style={styles.balanceBoxWalletNameContainer} onPress={this.handleOpenWalletListModal}>
+                <CryptoIcon currencyCode={currencyCode} sizeRem={1.5} walletId={walletId} />
                 <EdgeText style={styles.balanceBoxWalletName}>{walletName}</EdgeText>
                 <Ionicons name="chevron-forward" size={theme.rem(1.5)} color={theme.iconTappable} />
               </TouchableOpacity>
-              <CryptoIcon currencyCode={currencyCode} sizeRem={1.5} walletId={walletId} />
+              <TouchableOpacity onPress={this.handleMenu}>
```

Comment:
Suggest styling this TouchableOpacity containing the new settings icon with `alignSelf: 'center'` because the alignment is a bit strange to me in this current form:

<img width="300" alt="Screen Shot 2022-09-26 at 3 16 19 PM" src="https://user-images.githubusercontent.com/90650827/192390307-27f41492-853a-48f0-aa98-0ea3eaec21e7.png">